packages/driver/test/cypress/integration/commands/actions/type_spec.js

```diff
@@ -167,6 +167,18 @@ describe('src/cy/commands/actions/type', () => {
       })
     })
 
+    it('can type into element that redirects focus', () => {
+      cy.$$('div[tabindex]:first').on('focus', () => {
+        cy.$$('input:first').focus()
+      })
+
+      cy.get('div[tabindex]:first').type('foobar')
```

Comment:
what is the subject here? did it change to the new element? or stay as the old one?

Reply:
does not change